balloc/balloc.h

```diff
@@ -213,6 +214,26 @@ enum {
 	M0_BALLOC_BUDDY_LOOKUP_MAX = 10,
 };
 
+/*
+ * BALLOC_DEF_BLOCKS_PER_GROUP * (1 << BALLOC_DEF_BLOCK_SHIFT) = 128 MB -->
+ * which equals group size in ext4
+ */
+enum {
+	/** @todo should equal to size of HDD */
+	BALLOC_DEF_CONTAINER_SIZE	= 4096ULL * 32 * 1024 * 1000,
+	/** 4K Blocks */
+	BALLOC_DEF_BLOCK_SHIFT		= 12,
+	/**
+	 * Approximate number of groups. Exact number of groups can differ
+	 * depending on group size restrictions.
+	 */
+	BALLOC_DEF_GROUPS_NR            = 64,
+	/** Default db index count */
+	BALLOC_DEF_INDEXES_NR           = 1,
+	/** Used as minimal group size */
+	BALLOC_DEF_BLOCKS_PER_GROUP     = 524288,
```

Comment:
Looks like old values of BALLOC_DEF_BLOCKS_PER_GROUP was 32768, with this new value getting Integer overflow warning during compilation.
```
balloc/ut/balloc.c:46:78: error: integer overflow in expression [-Werror=overflow]
 #define GROUP_SIZE (BALLOC_DEF_CONTAINER_SIZE / (BALLOC_DEF_BLOCKS_PER_GROUP
```